minkindr/include/kindr/minimal/implementation/rotation-quaternion-inl.h

```diff
@@ -407,8 +409,9 @@ RotationQuaternionTemplate<Scalar>::operator*(
 
   // check if the multiplication has resulted in the quaternion no longer being
   // approximately normalized.
-  if (std::abs(result.squaredNorm() - static_cast<Scalar>(1.0)) >
-      EPS<Scalar>::normalization_value()) {
```

Comment:
maybe you can do a specialization on float here? or did we also have issues with doubles and normalization?